src/app/component/shared/detail-view/detail-view.component.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from "@angular/core";
+import { FormlyFieldConfig } from "@ngx-formly/core";
+import { AbstractModel } from "src/app/models/AbstractModel";
+
+@Component({
+  selector: "app-detail-view",
+  template: `
+    <div>
+      <div *ngFor="let field of fields">
```

Comment:
semantically, I'd say this is more reminiscent of a definition list https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl